src/v3.ts

```diff
@@ -154,27 +153,107 @@ export default function generateTypesV3(
     return output;
   }
 
-  if (rawSchema) {
-    const schemas = createKeys(propertyMapped, Object.keys(propertyMapped));
+  function transformPaths(paths: OpenAPI3Paths): string {
+    let output = "";
+    Object.entries(paths).forEach(([path, methods]) => {
+      output += `"${path}": {\n`;
+      Object.entries(methods).forEach(([method, responses]) => {
+        if (responses.description) output += comment(responses.description);
+        output += `"${method}": {\n`;
+
+        // handle parameters
+        if (responses.parameters) {
+          output += `parameters: {\n`;
+          const allParameters: Record<
+            string,
+            Record<string, OpenAPI3Parameter>
+          > = {};
+          responses.parameters.forEach((p) => {
+            if (!allParameters[p.in]) allParameters[p.in] = {};
+            // TODO: handle $ref parameters
```

Comment:
`parameters` can be a `$ref` rather than an object. The first cut doesn’t handle that (yet)